packages/client/screens/QRCodeScannerScreen.tsx

```diff
@@ -0,0 +1,241 @@
+import { Box, Center, Text, useToast, View } from "native-base";
+import React from "react";
+import { HomeStackScreenProps, SettingsStackScreenProps } from "../types";
+import { CompositeScreenProps } from "@react-navigation/native";
+import { BarCodeBounds, BarCodeScanner, Constants } from "expo-barcode-scanner";
+import { StyleSheet } from "react-native";
+import { getCurrencyTypeUILabel, isValidCurrencyAddress } from "@cryptify/common/src/utils/currency_utils";
+import { StatusBar } from "expo-status-bar";
+
+type Props = CompositeScreenProps<
+    HomeStackScreenProps<"QRCodeScannerScreen">,
+    SettingsStackScreenProps<"QRCodeScannerScreen">
+>;
+
+const initialBounds: BarCodeBounds = { origin: { x: 0, y: 0 }, size: { width: 0, height: 0 } };
+
+export default function QRCodeScannerScreen(props: Props) {
+    const { setFieldValue, fieldKey, currencyType } = props.route.params;
+
+    const toast = useToast();
+    const toastId = "qrCodeScannerToast";
+
+    const [scanned, setScanned] = React.useState(false);
+    const [bounds, setBounds] = React.useState<BarCodeBounds>(initialBounds);
+
+    const styles = getStyles(bounds);
+
+    const sleep = (t: number | undefined) => new Promise((res) => setTimeout(res, t));
+
+    const invalidAddressToast = (currencyType: string) => {
+        if (!toast.isActive(toastId)) {
+            toast.show({
+                id: toastId,
+                placement: "top",
+                duration: 2500,
+                render: () => {
+                    return (
+                        <Box style={styles.toastBox}>
+                            <Center>
+                                <Text
+                                    size={"footnote1"}
+                                    fontWeight={"semibold"}
+                                    color={"white"}
+                                    style={styles.toastText}
+                                    textAlign={"center"}
+                                >
+                                    This QR Code does not contain a valid {currencyType} wallet address.
+                                </Text>
+                            </Center>
+                        </Box>
+                    );
+                },
+            });
+        }
+    };
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    const handleBarCodeScanned = async ({ data, bounds }) => {
```

Comment:
I'm guessing the error was no implicit any, this should fix it

```suggestion
    const handleBarCodeScanned = async ({ data, bounds }: any) => {
```